app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.component.ts

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+import {compact, uniq, map} from 'lodash';
+import {module} from 'angular';
+import {Application} from '../../application/application.model';
+
+export const LOAD_BALANCER_FILTER = 'spinnaker.core.loadBalancer.filter.controller';
+
+let ngmodule = module('spinnaker.core.loadBalancer.filter.controller', [
+  require('./loadBalancer.filter.service.js'),
+  require('./loadBalancer.filter.model.js'),
+  require('../../filterModel/dependentFilter/dependentFilter.service.js'),
+  require('./loadBalancerDependentFilterHelper.service.js'),
+]);
+
+class LoadBalancerFilterCtrl {
+  public app: Application;
+  public availabilityZoneHeadings: any;
+  public regionHeadings: any;
+  public accountHeadings: any;
+  public providerTypeHeadings: any;
+  public stackHeadings: string[];
+  public tags: any[];
+
+  constructor(private $scope: angular.IScope,
```

Comment:
We've been using the pattern with `static get $inject() { return ['']; }` in addition to the constructor. I'm not sold on having it or not, but I do value consistency. We should decide on a pattern as a group and make it uniform.